packages/loader/lib/main.js

```diff
@@ -51,11 +59,10 @@ var mainContext = this;
 
       exports = seen[name] = {};
 
-      if (!registry[name]) {
+      if (!mod) {
         missingModule(name, referrerName);
```

Comment:
i think the error should be what the users input was, so this may want to be `missingModule(_name, referrerName)`